src/partition.rs

```diff
@@ -280,6 +280,10 @@ impl BlockSize {
 
   pub fn largest_chroma_tx_size(self, xdec: usize, ydec: usize) -> TxSize {
     let plane_bsize = self.subsampled_size(xdec, ydec);
+    if plane_bsize == BLOCK_INVALID {
+      panic!("invalid block size for this subsampling mode");
```

Comment:
Can we put this panic in subsampled_size instead? I'd love to get rid of the BLOCK_INVALID enum value.

Reply:
There are other areas in the code where the output of `subsampled_size()` is checked, so it is not doable without bigger changes.

Reply:
OK. We can leave that for some other time.

(or right now, I might do it)

Reply:
Sounds good.